packages/components/src/range-control/index.js

```diff
@@ -64,9 +64,12 @@ function RangeControl( {
 			parseFloat( newValue )
 		);
 	};
+
+	const initialFallbackValue = ! isUndefined( initialPosition ) ?
```

Comment:
We are using isFinite for checking currentInputValue, would it make sense to use isFinite to check initialPosition too?

Reply:
Ah yes! That works. Thank you!